hoth_api.yaml

```diff
@@ -319,5 +319,351 @@ components:
+          $ref: "#/components/schemas/GFEOverview"
+        contract_considerations: # DOW section 10 (and its subtasks)
+          $ref: "#/components/schemas/ContractConsiderations"
+        section_508_accessibility_standards_for_cloud_computing: # DOW section 11
+          $ref: "#/components/schemas/SensitiveInformation"
+    CurrentEnvironment:
+      description: "Part of DOW, encapsulates all data for 3. Background/Current Environment section."
+      properties:
+        current_environment_exists:
+          type: boolean
+        environment_instances:
+          type: array
+          items:
+            $ref: "#/components/schemas/EnvironmentInstance"
+          maxItems: 4000
+        additional_info:
+          type: string
+          maxLength: 1000
+    EnvironmentInstance:
+      description: "Part of DOW, represents a single Environment Instance"
+      properties:
+        instance_name:
+          type: string
+          maxLength: 100
+        classification_level:
+          $ref: "#/components/schemas/ClassificationLevel"
+        instance_location:
+          type: string
+          enum:
+            - CSP
+            - Hybrid
+            - Not in the cloud / On-premise
+        csp_region:
+          $ref: "#/components/schemas/TargetCloudServiceProviderName" #On the SNOW side, this is CSP_REGION a choice field with no options, so using TargetCSPName instead
+        performance_tier: # This is also an empty choice on the SNOW side, set as empty String until we get details
+          type: string
+        pricing_model:
+          type: string
+          enum:
+            - On-demand
+            - Pay-as-you-go
+            - Reserved
+        pricing_model_expiration:
+          type: string
+          format: date
+          example: "2022-07-01"
+        operating_system_licensing:
+          type: string
+          maxLength: 1000
+        number_of_vCPUs:
+          type: integer
+          maximum: 40
+        storage_type:
+          type: string # Empty choice on SNOW side, set as empty String until we get details
+        storage_amount:
+          type: integer
+          maximum: 40
+        storage_unit:
+          $ref: "#/components/schemas/StorageUnit"
+        memory_amount:
+          type: integer
+          maximum: 40
+        memory_unit:
+          $ref: "#/components/schemas/StorageUnit"
+        data_egress_monthly_amount:
+          type: integer
+          maximum: 40
+        data_egress_monthly_unit:
+          $ref: "#/components/schemas/StorageUnit"
+    ClassificationLevel:
+      description: "Part of DOW, Classification Level table fields"
+      properties:
+        classification:
+          type: string
+          enum:
+            - Unclassified
+            - Top Secret
+            - Secret
+        impact_level:
+          type: string
+          nullable: true
+          enum:
+            - IL2
+            - IL4
+            - IL5
+            - IL6
+            - null
```

Comment:
What if we just matched https://github.com/dod-ccpo/atat-csp-orchestration/blob/main/provisioning/atat_provisioning.yaml#L459 and made this not required (rather than nullable)?